cnd/src/db/tables.rs

```diff
@@ -8,8 +8,8 @@ use crate::{
         },
         Sqlite,
     },
-    halight, herc20, identity, lightning,
-    swap_protocols::{hbit, rfc003},
+    halight, hbit, herc20, identity, lightning,
+    swap_protocols::rfc003,
```

Comment:
`SecretHash` is in comit crate, we don't need this `rfc003` here.

Reply:
Given the simplicity of this change I just squashed it against the original commit. It's done now :D